package.json

```diff
@@ -195,6 +205,16 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.showFromDefaultExplorer",
+        "title": "ONE: Show Metadata",
+        "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.showFromOneExplorer",
+        "title": "ONE: Show Metadata",
```

Comment:
How about removing this `ONE:` if it's a context menu of ONE Explorer?

Reply:
ok, I got it! thank you!